internal/adapters/runtime/kubernetes/game_room.go

```diff
@@ -59,7 +63,9 @@ func (k *kubernetes) CreateGameRoomInstance(ctx context.Context, scheduler *enti
 	return instance, nil
 }
 
-func (k *kubernetes) DeleteGameRoomInstance(ctx context.Context, gameRoomInstance *game_room.Instance) error {
+func (k *kubernetes) DeleteGameRoomInstance(ctx context.Context, gameRoomInstance *game_room.Instance, reason string) error {
+	_ = k.createKubernetesEvent(ctx, gameRoomInstance.SchedulerID, gameRoomInstance.ID, "GameRoomDeleted", reason)
```

Comment:
It seems you're passing `reason` as the last arg which is `message string` in `createKubernetesEvent`. Shouldn't it be inverted with `GameRoomDeleted`?